docs/manifest.json

```diff
@@ -1045,6 +1045,11 @@
           "title": "Template ImagePullSecrets",
           "description": "Creating ImagePullSecrets for private registries",
           "path": "./guides/image-pull-secret.md"
+        },
```

Comment:
For manifest we can try to follow an alphabetical order. Thoughts?

Reply:
@matifali we will be re-ordering these soon under their own subsections, so we can leave this for another PR.

Reply:
Sure. Yes sectioning would be better. Do get me in the loop on the sectioning you are planning.